internal/pkg/validator/validator.go

```diff
@@ -18,8 +22,22 @@ type Validation struct {
 }
 
 func Validate(value interface{}, rules ...Validation) error {
+	return ValidateCtx(value, context.Background(), "dive", "", rules...)
+}
+
+func ValidateCtx(value interface{}, ctx context.Context, tag string, fieldName string, rules ...Validation) error {
 	// Setup
 	validate := validator.New()
+	enLocale := en.New()
+	universalTranslator := ut.New(enLocale, enLocale)
+	enTranslator, found := universalTranslator.GetTranslator("en")
+	if !found {
+		panic(fmt.Errorf("en translator was not found"))
+	}
+	err := enTranslation.RegisterDefaultTranslations(validate, enTranslator)
+	if err != nil {
+		panic(fmt.Errorf("translator was not registered: %w", err))
+	}
```

Comment:
Tento kod, az po `// Do` by som navrhoval dat do novej metody `new(rules ...Validation)`, ktora bude vytvarat nas nastaveny validator.

Reply:
Jméno `new` se mu nějak nelíbí 🤔
![CleanShot 2022-01-17 at 14 25 36](https://user-images.githubusercontent.com/215660/149777340-4214d63d-7b6c-4242-8f7a-7cbc9df3fc99.jpg)

Reply:
Pojmenuju teda `newValidator`

Reply:
jj, jasne, nedoslo mi, ze je to klucove slovo.